persistence/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/OffsetStore.scala

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
  */
-package com.lightbend.lagom.internal.persistence
+package com.lightbend.lagom.internal.javadsl.persistence
```

Comment:
The OffsetStore can probably be shared between Java and Scala, it's not public API.